modules/ocf_mirrors/templates/sync.service.erb

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Sync releases for the %i project
```

Comment:
I think `%i` should say something like `<%= @title %>` instead? systemd templates are not puppet templates.

Reply:
Yes, you are correct.